treeherder/etl/buildbot.py

```diff
@@ -171,6 +179,14 @@
         }
     },
     {
+        'regex': re.compile(r'(Yosemite|macosx64).*Stylo-disabled', re.IGNORECASE),
+        'attributes': {
+            'os': 'mac',
+            'os_platform': 'macosx64-stylo-disabled',
```

Comment:
This should be `osx-10-10-stylo-disabled` to match the form of the existing jobs above this one?